tests/ui/higher-ranked/leak-check/candidate-from-env-universe-err-1.rs

```diff
@@ -25,7 +26,6 @@ where
     // the leak check both candidates may apply and we prefer the
```

Comment:
This comment is out of date now?

Reply:
Well, maybe. Not sure actually.

Reply:
this comment still describes what we're testing 😁  could slightly reword it to make it clear that we do leak check here now